sklearn/feature_selection/_rfe.py

```diff
@@ -457,6 +458,24 @@ class RFECV(RFE):
         ``grid_scores_[i]`` corresponds to
         the CV score of the i-th subset of features.
 
+        .. deprecated:: 0.23
+        The `grid_scores_` attribute is deprecated in version 0.23 in favor
```

Comment:
```suggestion
          The `grid_scores_` attribute is deprecated in version 0.23 in favor
```

Reply:
This indentation and the next one will fix the 'unexpected unindent' sphinx warning (see [artifacts](https://105131-843222-gh.circle-artifacts.com/0/doc/_changed.html))